docs/json_data/sysfs_attributes_drm.json

```diff
@@ -154,7 +154,7 @@
             "format": "double",
             "units": "hertz",
             "writeable": false,
-            "alias": ""
+            "alias": "GPU_CORE_FREQUENCY_STATUS"
```

Comment:
Should we map this to RPS_ACT_FREQ instead? I think the difference would be if the driver told the gpu_chip to target some frequency and the gpu_chip doesn't achieve that frequency. In that case, this signal would show what the driver asked for but the other one would show what the tile achieved.

Reply:
The "ACT" signal is sometimes reported as zero but "CUR" seems to be more reliable.